diff -urN a/io/prometheus/client/BUILD.bazel b/io/prometheus/client/BUILD.bazel
--- a/io/prometheus/client/BUILD.bazel
+++ b/io/prometheus/client/BUILD.bazel
@@ -0,0 +1,9 @@
+load("@rules_proto//proto:defs.bzl", "proto_library")
+
+# keep
+proto_library(
+    name = "io_prometheus_client_proto",
+    srcs = ["metrics.proto"],
+    deps = ["@com_google_protobuf//:timestamp_proto"],
+    visibility = ["//visibility:public"],
+)
