diff --git a/capnp_cpp.bzl b/capnp_cpp.bzl
--- a/capnp_cpp.bzl
+++ b/capnp_cpp.bzl
@@ -1,10 +1,10 @@
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 
-URL = "https://github.com/capnproto/capnproto/tarball/a7654b2041731d0ee323aa2bdb7975c7ab903f5e"
-STRIP_PREFIX = "capnproto-capnproto-a7654b2/c++"
-SHA256 = "f502ac3ada732800929706f3d6dacb9bcf3639d3bffde6387f4dd5e0e11984f5"
+URL = "https://github.com/capnproto/capnproto/tarball/911e53d67841687afe9a349fd8c0d39fe024515a"
+STRIP_PREFIX = "capnproto-capnproto-911e53d/c++"
+SHA256 = "a87651d1772c138643e1fc28ca0cbc8eda0445ca265bc200c083c31a75919386"
 TYPE = "tgz"
-COMMIT = "a7654b2041731d0ee323aa2bdb7975c7ab903f5e"
+COMMIT = "911e53d67841687afe9a349fd8c0d39fe024515a"
 
 def _capnp_cpp(ctx):
     http_archive(
