diff --git a/build/deps/gen/deps.MODULE.bazel b/build/deps/gen/deps.MODULE.bazel
--- a/build/deps/gen/deps.MODULE.bazel
+++ b/build/deps/gen/deps.MODULE.bazel
@@ -134,14 +134,7 @@
 bazel_dep(name = "tcmalloc", version = "0.0.0-20250927-12f2552")
 
 # workerd-cxx
-http.archive(
-    name = "workerd-cxx",
-    sha256 = "fbba1b102b2c4fe879b2f610d7e94ceda6beceac3d57a27196482ce3e9536b50",
-    strip_prefix = "cloudflare-workerd-cxx-c677ef5",
-    type = "tgz",
-    url = "https://github.com/cloudflare/workerd-cxx/tarball/c677ef53092a8425ce9f059074441fdb1b7c1ed3",
-)
-use_repo(http, "workerd-cxx")
+bazel_dep(name = "workerd-cxx", version = "0.0.0")
 
 # zlib
 bazel_dep(name = "zlib")
