diff --git a/build/deps/v8.MODULE.bazel b/build/deps/v8.MODULE.bazel
--- a/build/deps/v8.MODULE.bazel
+++ b/build/deps/v8.MODULE.bazel
@@ -75,6 +75,9 @@
     build_file = "@v8//:bazel/BUILD.icu",
     commit = "ee5f27adc28bd3f15b2c293f726d14d2e336cbd5",
-    patch_cmds = ["find source -name BUILD.bazel | xargs rm"],
+    patch_cmds = [
+        "find source -name BUILD.bazel | xargs rm",
+        "python3 -c 'from pathlib import Path; p=Path(\"BUILD.bazel\"); s=p.read_text(); p.write_text(\"load(\\\"@rules_cc//cc:cc_library.bzl\\\", \\\"cc_library\\\")\\n\" + s if \"cc_library.bzl\" not in s else s)'",
+    ],
     patch_cmds_win = ["Get-ChildItem -Path source -File -Include BUILD.bazel -Recurse | Remove-Item"],
     remote = "https://chromium.googlesource.com/chromium/deps/icu.git",
 )
